test/lib/functions.test.js

```diff
@@ -213,7 +213,7 @@ describe('FlexFunctions', () => {
     it('should include a body', (done) => {
       const taskName = quickRandom();
       const task = sampleTask(taskName);
-      functions.register(taskName, (context, complete) => complete({ foo: 'bar' }).ok().next());
+      functions.register(taskName, (context, complete) => complete({ foo: 'bar' }).ok().done());
```

Comment:
Older method is still valid, and in fact I think it's a useful syntactic shortcut.  But we'll document the `setBody` as the first primary way, and the `complete` as the shortcut.